examples/subplots_axes_and_figures/plot_parasite_axes.py

```diff
@@ -0,0 +1,31 @@
+"""
+=====================================================
+Plotting datasets in different units on the same axes
+=====================================================
+
+This example showcases how to plot two different related datasets with
+different units on the same axes. These are often called "parasite" axes,
+as the second axis is an extension of the first.
```

Comment:
Actually, the terminology is not quite consistent. The title says "same axes" which is ok to attract attention and is what the user wants to do semantically.

Proposal:
This example showcases how to plot two different related datasets with
different y-scales on top of each other. To achieve this, we create a second 
Axes that shares the x-axis with the first Axes using `~Axes.twinx()`.
Such Axes are often called "parasite" axes.
+as the second axis is an extension of the first.

Reply:
I like it! just pushed your changes to nelle's branch